libcontainer/seccomp/seccomp_linux.go

```diff
@@ -42,7 +42,14 @@ func InitSeccomp(config *configs.Seccomp) (int, error) {
 
 	// Ignore the error since pre-2.4 libseccomp is treated as API level 0.
 	apiLevel, _ := libseccomp.GetAPI()
+	const writeSyscall = "write"
+	writeAllowed := false
 	for _, call := range config.Syscalls {
+		// If write is in the allowed list of syscalls, then we can also notify on the defaultAction.
+		if call.Name == writeSyscall {
+			writeAllowed = syscallIsAlwaysAllowed(call, config.Architectures)
```

Comment:
Hmm, if for some reason the syscall entry is repeated for write, what should we do? I don't know what libseccomp will do, nor if that behavior is guaranteed.